src/utilities/ContextBuilder.js

```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+import { v4 as uuid } from 'uuid';
+
+const getContext = (action) => ({
```

Comment:
Context should not be built in frontend. Its in BAP

Reply:
In Search flow, it interacts with the Gateway. But in this scenario we have to send it directly from BAP to BPP